### ISSUE TITLE
Make "next/previous" slide translatable strings; right now, they are hard-coded 

### ISSUE THREAD

**Issue by beatricelucaci** [WPChill/strong-testimonials#430]
**Is your feature request related to a problem? Please describe.**
The “next” link slug is next-slide. Even though the plugin settings call it a slideshow, for Accessibility it should be next-testimonial.
How to change this?

1. https://wordpress.org/support/topic/change-name-of-next-slide-to-next-testimonial/
2. https://github.dev/WPChill/strong-testimonials/blob/8a33806752e3fe3d1c97541a5942065b3d6b0903/public/js/lib/strongslider/jquery-strongslider.js#L1008
3. https://github.dev/WPChill/strong-testimonials/blob/8a33806752e3fe3d1c97541a5942065b3d6b0903/public/js/lib/strongslider/jquery-strongslider.js#L1033-L1034

**Comment by cristianraiber** [comment]
@TeoAlex - use `wp_localize_script` to pass the strings to the JS script
